### PR TITLE
Initial version of translation process description in English

### PR DESCRIPTION
**Pull request by Gaket**
This is a draft of the translation process as discussed [here](https://github.com/tlbootcamp/tlroadmap/pull/121)

Will add Russian "source" document when we agree on the structure and meaning of the English version.

### CODE REVIEW

File: translations/process/translation-process.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Working on translations
```
Comment (vvscode) [inline translations/process/translation-process.md]: what about https://gitlocalize.com/ ?

Comment (Gaket) [reply translations/process/translation-process.md]: Looks interesting. I've installed it in my fork, but it allows to only add a translation a block by block, so I haven't migrated my translation into it.

@etolstoy, what do you think about the tool? It looks neat and it's free.

Comment (etolstoy) [reply translations/process/translation-process.md]: Wow, it looks really awesome. Let's try to integrate it!

File: translations/process/translation-process.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+This project introduces a simple translations versioning system. We want to have a simple way of finding a diff to be translated. Similar to decisions done with the CAP theorem, we sacrifice consistency (applying eventual consistency) to higher availability (documents in different languages). 
+
+## Creating a new translation
+
+To facilitate the process of finding and translating the diff, we add a commit hash of the doc in Russian, considered as the base language, to the end of the translated page, section "Translation notes. Here is [an example](README-en.md)
+
+## Updating existing translation
+
+If a translation simply improves wording or fixes small errors no additional changes required.
```
Comment (php-coder) [inline translations/process/translation-process.md]: no additional changes required -> no additional changes are required
?

File: translations/process/translation-process.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+If a translation aims to mirror some changes in the original file, the whole diff between the old and new translations should be translated. 
+1. Find the [diff](https://git-scm.com/docs/git-diff) between the already translated version and the version that is going to be translated.
+1. Translate the changes, add improvements to existing parts if needed.
+1. Update the "Translation notes" section in the updated translation file, changing the commit link to the original file version commit.
+
+## Creating a doc that is not in Russian
+
+We want to achieve a unidirectional flow of translations and chose Russian as a base language of the repo. As a result, we are asking everyone to kindly add Russian version to any document.
```
Comment (php-coder) [inline translations/process/translation-process.md]:
```suggestion
We want to achieve a unidirectional flow of translations and chose Russian as a base language of the repo. As a result, we ask everyone to kindly add a Russian version to any document.
```

**Comment by etolstoy** [comment]
I'm closing this PR for now. We decided to complete the russian version first, and after it think about translating to english.

**Pull request closed without merging**